client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -127,6 +130,14 @@ public static ProxySelector createProxySelector(ProxyConfiguration proxyConfig)
         switch (proxyConfig.type()) {
             case DIRECT:
                 return fixedProxySelectorFor(Proxy.NO_PROXY);
+            case FROM_ENVIRONMENT:
+                String defaultEnvProxy = System.getenv(ENV_HTTPS_PROXY);
+                Preconditions.checkNotNull(
```

Comment:
Put this on one line `String defaultEnvProxy = Preconditions.checkNotNull...`

Reply:
👍